### PR TITLE
sbt 1.3.2

### PR DESCRIPTION
**Pull request by takayahilton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,11 +5,11 @@ script: sbt ^^$TRAVIS_SBT_VERSION scripted
 matrix:
   include:
     - env: TRAVIS_SBT_VERSION="0.13.18"
-    - env: TRAVIS_SBT_VERSION="1.2.8"
+    - env: TRAVIS_SBT_VERSION="1.3.0"
 
 before_cache:
   - find $HOME/.sbt -name "*.lock" | xargs rm
-  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs --no-run-if-empty rm
```
Comment (gslowikowski) [inline .travis.yml]: Why `--no-run-if-empty` switch was added? Were there any problems?

File: .travis.yml [hunk 2]
```diff
@@ -5,11 +5,11 @@ script: sbt ^^$TRAVIS_SBT_VERSION scripted
 matrix:
   include:
     - env: TRAVIS_SBT_VERSION="0.13.18"
-    - env: TRAVIS_SBT_VERSION="1.2.8"
+    - env: TRAVIS_SBT_VERSION="1.3.0"
```
Comment (jiminhsieh) [inline .travis.yml]: 1.3.2 was just released.

File: .travis.yml (hunk 1 above)
Comment (takayahilton) [reply .travis.yml]: `sbt 1.3.x` uses Courseier to resolve dependencies.
I think that `ivydata-*.properties` is not made in `sbt 1.3.x`.

**Comment by Martin456** [comment]
Hello guys, would it be possible to move forward with this MR. Sbt 1.3.7 was published already.

**Pull request closed without merging**